pkg/morph/client/notary.go

```diff
@@ -510,6 +510,14 @@ func (c *Client) notaryCosigners(ir []*keys.PublicKey, committee bool) ([]transa
 		AllowedGroups:    c.signer.AllowedGroups,
 	})
 
+	// then we have invoker signature
+	s = append(s, transaction.Signer{
+		Account:          hash.Hash160(c.acc.GetVerificationScript()),
```

Comment:
Please, use `c.acc.ScriptHash()` for that.

Reply:
Can you give a reference for that in latest neo-go? Couldn't find `ScriptHash()` implementation on account structure.

Reply:
Oh, sorry, that's my bad, there's no such API defined on `Account`. There's only `acc.PrivateKey().GetScriptHash()`, but it's better to keep ` hash.Hash160(c.acc.GetVerificationScript())` then.